web/controllers/v1/relay_controller.ex

```diff
@@ -1,28 +1,23 @@
 defmodule Cog.V1.RelayController do
   use Cog.Web, :controller
 
-  alias Cog.Models.Relay
-  alias Cog.Queries
-  alias Cog.Repo
+  alias Cog.Repository.Relays
```

Comment:
What do you think about changing this alias to at least `Repository.Relays`? I'm concerned about readers getting potentially confused by just seeing `Relays` and thinking of `Cog.Relay.Relays`.